### DISCUSSION TITLE
Does Immich sync changes after a photo is edited (Android)?

### DISCUSSION THREAD

**Discussion by PHLAK** [immich-app/immich#5560] [Q&A]
On Android,

  1. Take a photo
  2. Wait fit it to sync to the Immich server
  3. Edit the photo using Google Photos
  4. Save (in place, not a copy)

### Results

The edited photo shows up as a separate image in the Immich app on my phone, however, the changes aren't synced to the server.

Is this the intended behavior? Is there a way to sync the changes?

**Comment by michaelkebe** [comment]
This should be a bug report.
